src/provider/controller/ContextHandler.ts

```diff
@@ -20,7 +20,7 @@ export class ContextHandler {
     private readonly _model: Model;
 
     constructor(
-        @inject(Inject.API_HANDLER) apiHandler: APIHandler<APIFromClientTopic>,
+        @inject(Inject.API_HANDLER) apiHandler: APIHandler<APIFromClientTopic>, // eslint-disable-line @typescript-eslint/indent
```

Comment:
Huh. Is this linting rule changes with the updated service-tooling? Why was the eslint exception not needed before?